test/client/desktop/ApplicationWindowTest.js

```diff
@@ -214,24 +260,33 @@ o.spec("ApplicationWindow Test", function () {
 		// instances
 		const wmMock = n.mock('__wm', wm).set()
 
+		const themeManagerMock = n.mock('__themeManager', themeManager).set()
+
 		return {
 			electronMock,
 			electronLocalshortcutMock,
 			desktopTrayMock,
 			langMock,
 			wmMock,
+			themeManagerMock
 		}
 	}
 
-	o("construction", function () {
-		const {electronMock, wmMock, electronLocalshortcutMock} = standardMocks()
+	o("construction", async function () {
+		const {electronMock, wmMock, electronLocalshortcutMock, themeManagerMock} = standardMocks()
 
-		const w = new ApplicationWindow(wmMock, desktopHtml, icon, electronMock, electronLocalshortcutMock, "dictUrl")
+		const w = new ApplicationWindow(wmMock, desktopHtml, icon, electronMock, electronLocalshortcutMock, themeManagerMock, "dictUrl")
 
 		o(electronMock.BrowserWindow.mockedInstances.length).equals(1)
 		const bwInstance = electronMock.BrowserWindow.mockedInstances[0]
+		// We load some things async before loading URL so we wait for it. __loadedUrl comes from our mock
+		await bwInstance.__loadedUrl.promise
 		o(bwInstance.loadURL.callCount).equals(1)
-		o(bwInstance.loadURL.args[0]).equals('file:///path/to/app/desktophtml')
+		const theme = await themeManager.getCurrentThemeWithFallback()
+		const themeJson = JSON.stringify(theme)
+		const query = new URLSearchParams({theme: themeJson})
+		o(bwInstance.loadURL.args[0])
+			.equals(`file:///path/to/app/desktophtml?${query.toString()}`)
```

Comment:
Maybe hardcode the string here, it should be just two colors, no?

Reply:
I think the problem is that it's json inside query string and you can't read it well anyway

Reply:
`file:///path/to/app/desktophtml?theme=%7B%22themeId%22%3A%22light-fallback%22%2C%22content_bg%22%3A%22%23ffffff%22%2C%22header_bg%22%3A%22%23ffffff%22%7D`

:upside_down_face: